config/initializers/rswag-ui.rb

```diff
@@ -5,6 +5,7 @@
   # JSON endpoint and the second is a title that will be displayed in the document selector
   # NOTE: If you're using rspec-api to expose Swagger files (under swagger_root) as JSON endpoints,
   # then the list below should correspond to the relative paths for those endpoints
-  c.swagger_endpoint '/api-docs/current/swagger.json', 'API V2 Docs'
```

Comment:
Why remove V2?

Reply:
I think the earlier text was a typo, it actually has the `current` link which I updated to V3

Reply:
Also, I'm not sure what this doing exactly, the V2 docs do still show up under /api-docs

Reply:
Meh, ok 😅